### PR TITLE
Document modern forwarding with Paper

### PR DESCRIPTION
**Pull request by mdcfe**
Add details for using modern forwarding with Paper (see PaperMC/Paper#1557).

### CODE REVIEW

**Comment by mdcfe** [comment]
@astei Now that PaperMC/Paper#1557 has been merged, is there anything that needs altering in this PR?

File: users/player-info-forwarding.rst [hunk 1]
```diff
@@ -17,7 +17,22 @@ forwarding player information to your servers:
 Configuring modern forwarding
 -----------------------------
 
-Currently, no server implementations support Velocity's modern forwarding.
+Currently, only recent versions of Paper 1.13.1+ support Velocity's modern forwarding.
```
Comment (astei) [inline users/player-info-forwarding.rst]: Include build 377 or higher

File: users/player-info-forwarding.rst [hunk 2]
```diff
@@ -17,7 +17,22 @@ forwarding player information to your servers:
...
+setting to a unique secret. You then need to ensure your server is properly configured to
+use modern Velocity forwarding.
+
+Paper
+^^^^^
+
+To allow Paper to understand the forwarded player data, in your ``paper.yml``, set
+``settings.velocity-support.enabled`` to ``true`` and ``settings.velocity-support.secret``
+to match the secret in your Velocity config. To run your server in online mode, you can
+optionally set ``settings.velocity-support.online-mode`` to ``true``. Once you're done
```
Comment (astei) [inline users/player-info-forwarding.rst]: `Also, you must set settings.velocity-support.online-mode to the online-mode setting in velocity.toml`

**Review by astei — CHANGES_REQUESTED** [review]

File: users/player-info-forwarding.rst (hunk 1 above)
Comment (mdcfe) [reply users/player-info-forwarding.rst]: Should I put the information about which build supports it here, or under the `Paper` subheading? It's Paper-specific, so I don't think it necessarily belongs immediately below the `Configuring modern forwarding` heading.

Comment (astei) [reply users/player-info-forwarding.rst]: Put it in this paragraph.

File: users/player-info-forwarding.rst [hunk 3]
```diff
@@ -17,7 +17,22 @@ forwarding player information to your servers:
 Configuring modern forwarding
 -----------------------------
 
-Currently, no server implementations support Velocity's modern forwarding.
+Currently, only builds 337 and above of Paper 1.13.1+ support Velocity's modern forwarding.
```
Comment (astei) [inline users/player-info-forwarding.rst]: https://papermc.io/ci/job/Paper-1.13/377/ is the build that introduced Velocity support.

**Review by astei — CHANGES_REQUESTED** [review]

**Review by astei — APPROVED** [review]

**Pull request merged**